v0/destinations/monetate/data/MONETATEPage.json

```diff
@@ -0,0 +1,37 @@
+[
+    {
+        "destKey": "events",
+        "sourceKeys": ["properties.page", "context.page"],
+        "metadata": {
+            "targetType": "Array",
+            "isTargetTypePrimitive": false,
```

Comment:
may not require another key and find it from the "targetType"
Can discuss it in our broader refactoring plan